modules/test/playwright/tests/commerce/commerce-order-content-web/pendingOrder.spec.ts

```diff
@@ -0,0 +1,148 @@
+            case 'product':
+                await apiHelpers.headlessCommerceAdminCatalog.deleteProduct(
+                    item.id
+                );
+
+                break;
+            case 'site':
+                await apiHelpers.headlessSite.deleteSite(item.id);
+
+                break;
+            default:
+                break;
+        }
+    }
+});
+
+test('Edit pending order item with UOM', async ({
+    apiHelpers,
+    applicationsMenuPage,
+    commerceLayoutsPage,
+    commerceMiniCartPage,
+    pendingOrdersPage,
+    page,
+}) => {
+    await apiHelpers.featureFlag.updateFeatureFlag('COMMERCE-9599', true);
+
+    const site = await apiHelpers.headlessSite.createSite('Edit pending order');
+
+    data.push({id: site.id, type: 'site'});
+
+    const channel = await apiHelpers.headlessCommerceAdminChannel.postChannel({
+        name: 'Edit pending order Channel',
+        siteGroupId: site.id,
+    });
+
+    data.push({id: channel.id, type: 'channel'});
+
+    const catalog = await apiHelpers.headlessCommerceAdminCatalog.postCatalog({
+        name: 'Edit pending order Catalog',
+    });
+
+    data.push({id: catalog.id, type: 'catalog'});
+
+    const product1 = await apiHelpers.headlessCommerceAdminCatalog.postProduct({
+        catalogId: catalog.id,
+        name: {en_US: 'Product1'},
+    });
+
+    data.push({id: product1.productId, type: 'product'});
+
+    const product1Skus = await apiHelpers.headlessCommerceAdminCatalog
+        .getProduct(product1.productId)
+        .then((product) => {
+            return product.skus;
+        });
+
+    const sku1 = product1Skus[0];
+
+    const sku1SkuUnitOfMeasure =
+        await apiHelpers.headlessCommerceAdminCatalog.postSkuUnitOfMeasure(
+            sku1.id,
+            {
+                incrementalOrderQuantity: 3,
+                name: {en_US: 'Box'},
+                primary: true,
+                priority: 1,
+                rate: 1,
+            }
+        );
+
+    const sku2SkuUnitOfMeasure =
+        await apiHelpers.headlessCommerceAdminCatalog.postSkuUnitOfMeasure(
+            sku1.id,
+            {
+                incrementalOrderQuantity: 2,
+                name: {en_US: 'Package'},
+                priority: 2,
+                rate: 0.5,
+            }
+        );
+
+    await applicationsMenuPage.goToSite('Edit pending order');
+
+    await commerceLayoutsPage.goToPages(false);
+    await commerceLayoutsPage.createWidgetPage('Pending Orders Page');
+    await commerceLayoutsPage.goToPages(false);
+    await commerceLayoutsPage.changeCurrentTheme(
+        'Select Minium By Liferay, Inc.'
+    );
+    await commerceLayoutsPage.siteHomePageLink.click();
+
+    await commerceMiniCartPage.miniCartButton.click();
+    await commerceMiniCartPage.searchProductsInput.fill(sku1.sku);
+
+    await page
+        .getByRole('menuitem', {name: `${sku1.sku}`})
+        .click();
+
+    await commerceMiniCartPage.quickAddToCartButton.click();
```

Comment:
Can we create a function in the commerceMiniCartPage to add a product with the quickAddToCart by sku? I think it will be a useful method for all the tests

Reply:
And another question: why are you using minicart and not api to create an order with the sku?

Reply:
I've copied it from another test

Reply:
but it makes sense